homeassistant/components/onewire/const.py

```diff
@@ -37,6 +37,20 @@
 }
 DEVICE_SUPPORT_SYSBUS = ["10", "22", "28", "3B", "42"]
 
+DEVICE_SUPPORT_PRECISION_MAPPING = ["28"]
```

Comment:
Let's make this generic:
```suggestion
DEVICE_SUPPORT_OPTIONS = ["28"]
```